src/test/java/com/checkmarx/ast/ScanTest.java

```diff
@@ -19,6 +23,22 @@ void testScanShow() throws Exception {
         Assertions.assertEquals(scanList.get(0).getId(), scan.getId());
     }
 
+    @Test
+    void testScanVorpalSuccessfulResponse() throws Exception {
+        ScanResult scanResult = wrapper.ScanVorpal("src/test/resources/csharp-file.cs", true);
+        Assertions.assertNotNull(scanResult.getRequestId());
+        Assertions.assertTrue(scanResult.isStatus());
+        Assertions.assertNull(scanResult.getError());
+    }
+
+    @Test
+    void testScanVorpalFailureResponse() throws Exception {
+        ScanResult scanResult = wrapper.ScanVorpal("src/test/resources/csharp-file.cs", false);
+        Assertions.assertEquals("1111", scanResult.getRequestId());
```

Comment:
please have a case with a real error as not a good file name. 
If you have tests with files please have the real file 
Please validate the message